Packages/MIES/MIES_DAEphys_Macro.ipf

```diff
@@ -952,6 +952,7 @@ Window DA_Ephys() : Panel
 	ValDisplay ValDisp_DataAcq_SamplingInt,limits={0,0,0},barmisc={0,1000}
 	ValDisplay ValDisp_DataAcq_SamplingInt,value= _NUM:0
 	SetVariable SetVar_Sweep,pos={210.00,531.00},size={75.00,35.00},bodyWidth=75,disable=1,proc=DAP_SetVarProc_NextSweepLimit
+	SetVariable SetVar_Sweep,help={"The up/down arrows allow to adjust the current and next sweep number.<p style=\"color:Red;\"><strong>WARNING!</strong></p>This is an advanced feature which permanently removes already acquired data."}
```

Comment:
@t-b instead of: 
```
This is an advanced feature which permanently removes already acquired data
```

how about:

```
This is an advanced feature that overwrites acquired data. Data will be permanently lost.
```

Reply:
Done.